src/Web/Sfa.Das.Sas.Shared.Components/Sfa.Das.Sas.Shared.Components.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>SFA.DAS.SAS.Shared.Components</id>
-    <version>1.1.0.18-alpha-basket</version>
+    <version>1.1.0.23-alpha-basket</version>
```

Comment:
Has there been any changes to the shared components project to need a version increase?